Packs/Tidy/Integrations/Tidy/Tidy.yml

```diff
@@ -18,6 +18,11 @@ configuration:
   name: password
   required: true
   type: 4
+- display: SSH Key
+  hidden: false
+  name: ssh_key
+  required: false
+  type: 9
```

Comment:
```suggestion
  type: 15
```

Reply:
Why 15? it's for single select

Reply:
14*

Reply:
I think it should stay 9. 
It shouldn't be visible. We're using a pem key here.

Reply:
14 is long text encrypted (not visible) 
why do we need authentication param for ssh key?

Reply:
The pycharm plugin showed me it's a short text so I thought it's regular. Shouldn't it be 12? 
I think 14 is for short encrypted and 12 is for long (checked in with teams integration).

Reply:
12 is long text (not encrypted), 14 is long text encrypted